cmp/cmpopts/ignore.go

```diff
@@ -143,3 +144,58 @@ func isExported(id string) bool {
 	r, _ := utf8.DecodeRuneInString(id)
 	return unicode.IsUpper(r)
 }
+
+// IgnoreSliceElements returns an Option that ignores elements of []V.
+// The discard function must be of the form "func(T) bool" which is used to
+// ignore slice elements of type V, where V is assignable to T.
+func IgnoreSliceElements(discard interface{}) cmp.Option {
```

Comment:
discardFunc, to make it clear this is a function?

Reply:
Done. (adjusted other functions in the same package for consistency)